page "Accept Terms and Conditions

GabrielaRsanches opened the pull request.

Commit:
feat: crated css file to change the margin and finished html of the task

albnunes commented on apps/atqr/web/src/pages/termsAndCondition/termsAndCondition.html:
```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```
Use pug files.

rairezende replied:
@albnunes @yfernandes I'm taking care of this one

rairezende committed:
rai turned the html and css into pugs

feat: updated pug and css

feat: updated pug and css

chore(deadline monitor and challenge status): fix things in the deadlinemonitor to work

refactor(challenge created web page): cleaning the code and the style of the page

feat(husky,package.json.): check if the code in the staged area is within the project's standard

Check if the code in the staged area is within the project's standard.

Update challenge.entity.ts

chore: missing bracket

Update challenge.entity.ts

method updateStatus has been deleted. I added it again

refactor: refactor

 ts files e pug files para envio dos e-mails

Co-authored-by: Raquel <raquel@toki.life>

.pug youWereChallenged

img folder

created imgs folder

temporary teste folder for templates

temporary teste folder in templates

perf(emails templates): did a base template for all emails

import pug in ts file and compile

sync commit

html merged

mixin base template

base-style

naming improve

deadlineEmail template update

no more necessary folders deleted

email pics updated

added 3 email templates

templates index ts commented

templates updated n width to 740

hoodie gif deleted

chore(template emails): made templates for the emails we send

created btn mixin and added to templates

added btn mixin to templates

refactor(template emails): cleaning the code and exporting the classes from email

pics renamed

feat: did email classes that were missing

added btn-container and updated pug files

css and pug files improved

repo maint. cmmd updated and index n supCon upd

player and challenge repo entity created

IChallengeRepository's import path updated

fix(lib/domain): removes js files on domain

Probably an artefact of a stray TSC

feature: interface and symbols for repository finished

defined Dependency Injection in app module

inject the symbols in repositories

implemented interfaces in repositories and updated import's

updates in challenge and player indexes

fix: fix the imports in interfaces

Alberto/atqr 94: Adds Updates Challenge Endpoint (#32)

* refactor: entity method that updates status

* feature: I used the notification service as a base to notify the player

* refactor: redid the endpoint more correct than before

* fix(api): fixes minor mistakes created after rebasing

Co-authored-by: Yago Fernandes <yagoalmeida@gmail.com>

chore(domain): adds index files to domain

Alberto/atqr 92: Refactor Update challenge methods (#31)

* feature: create a global dto to endpoint patch

* implement the methods in endpoint

* the methods in front are updated

* refactor: Passed the logic to a middleware

* formatted document

* feat(controller): added a route for each method in controller

added a different patch route for method in controller

* refactor(atqr api): adds api indexes and absolutes paths to api and Tsconfig

* refactor: i removed useless imports and cleaned the code

* fix(api/dto): renames Update Challenge Dto

* chore(api): fixes spelling mistakes and other minor fixes

Co-authored-by: Yago Fernandes <yagoalmeida@gmail.com>

refactor(terms-style.scss): changed to css to scss

build(yarn run for catch tools): runed yarn

feat: changed kebab cases for camelCases and ajusted mixin terms's indentation

refactor(web): pages are now running and without compiled files

albnunes commented on apps/atqr/web/src/pages/termsAndCondition/termsConditions.pug:
```diff
@@ -0,0 +1,83 @@
+
+
+    //- Favicon
+    link(rel='shortcut icon' type='image/x-icon' href='../../assets/images/favicon.png')
```
These imports can be made by a mixin that exists in the index folder

albnunes commented on apps/atqr/web/src/pages/challenge/challenge.js:
```diff
@@ -0,0 +1,78 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```
this file can't be here. exclude

albnunes commented on apps/atqr/web/src/pages/termsAndCondition/termsConditions.pug:
```diff
@@ -0,0 +1,83 @@
+  script(src='../../assets/js/vendor/modernizr-3.11.7.min.js')
+
+  script(src='../../assets/js/all.js')
+  script(src='../../assets/js/back-to-top.js')
```
same comment above

albnunes reviewed (changes requested):
Make the requested changes

albnunes reviewed (dismissed):
it's ok
